packages/ui-core/static/styles/gauzy/_gauzy-skeleton.scss

```
@@ -0,0 +1,87 @@
+// ── Data-table loading skeleton ──────────────────────────────────────────────
+//
+// `angular2-smart-table` renders
+//     <tr *ngIf="grid.getRows().length == 0"><td [colspan]="n">{{ noDataMessage }}</td></tr>
```

Stylelint (17.14.1)

[error] Unknown word noDataMessage (CssSyntaxError)

(CssSyntaxError)